ssa/manager_wait.go

```diff
@@ -136,6 +136,12 @@ func (m *ResourceManager) WaitForSet(set object.ObjMetadataSet, opts WaitOptions
 			errs = append(errs, builder.String())
 		case errors.Is(ctx.Err(), context.DeadlineExceeded) && lastStatus[id].Status != status.CurrentStatus:
 			var builder strings.Builder
+			if utils.IsSuspended(lastStatus[id].Resource) {
```

Comment:
Is it safe to do this? If I have several resources deployed by a `kustomization`, and I want them all `healthy` before triggering another dependent `kustomization`, skipping a suspended kustomization might have unforseen consequences.